pkgs/desktops/pantheon/desktop/gala/default.nix

```diff
@@ -70,6 +70,7 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./plugins-dir.patch
+    ./use-new-notifications-default.patch
```

Comment:
Can't we use the override here as well?

Reply:
It was weird. When gala would crash it would revert to not using the new notifications if I didn't do this.

Reply:
Yeah, it will need to be done here too since if gala somehow loses the environment, it will fall back to the one from wrappers. But I meant that we could try to re-use the overrides, since that is what they are for – affecting the default value in the compiled schemas.

Reply:
Ahh, you mean put `use-new-notifications` in `20_elementary.pantheon.wm.gschema.override`.?

Reply:
Possibly. Though it probably does not matter.